qiskit/visualization/matplotlib.py

```diff
@@ -26,6 +26,8 @@
 import numpy as np
 
 try:
+    import matplotlib
+    matplotlib.use('ps')
```

Comment:
This is the same problem as I pointed out in #3002 using `use()` in the actual visualizer code goes too far. We need the `use()` usage to be isolated to test code. The dependency on a specific backend is the image comparison tests, not the actual visualizer.

Reply:
Moving it to `test/python/visualization/visualization.py`